src/Portage/Metadata.hs

```diff
@@ -7,62 +7,64 @@ Functions and types related to @metadata.xml@ processing
 -}
 module Portage.Metadata
         ( Metadata(..)
-        , metadataFromFile
-        , pureMetadataFromFile
+        , readMetadataFile
+        , parseMetadataXML
         , stripGlobalUseFlags -- exported for hspec
         , prettyPrintFlags -- exported for hspec
         , prettyPrintFlagsHuman
-        , makeDefaultMetadata
-        , makeMinimalMetadata
+        , printMetadata
+        , minimalMetadata
         ) where
 
 import qualified AnsiColor as A
 
+import Control.Monad
 import qualified Data.List       as L
 import qualified Data.Map.Strict as Map
+import qualified Data.Set        as S
 import qualified Data.Text       as T
 import qualified Data.Text.IO    as T
 
 import Text.XML.Light
 
+import Portage.EBuild as EBuild
+import Portage.Metadata.RemoteId
+
+type UseFlags = Map.Map String String
+
 -- | A data type for the Gentoo-specific @metadata.xml@ file.
 -- Currently defines functions for the maintainer email and
 -- USE flags and their descriptions.
 data Metadata = Metadata
       { metadataEmails :: [String] -- ^ This should /always/ be [\"haskell@gentoo.org\"].
```

Comment:
If this should always be `["haskell@gentoo.org"]`, why do we have it as a variable?

Reply:
I'm not sure. This was from before my time.